### ISSUE TITLE
`Error: Cannot find module 'browserify'`

### ISSUE THREAD

**Issue by paulmenzel** [clutchski/coffeelint#562]
Installing the Node.js packages with `npm install`, the installation currently fails with the error below.

```
$ npm install
[…]
> coffeelint@1.15.1 install /home/app/node_modules/gulp-coffeelint/node_modules/coffeelint
> cake install

module.js:327
    throw err;
    ^

Error: Cannot find module 'browserify'
  at Function.Module._resolveFilename (module.js:325:15)
  at Function.Module._load (module.js:276:25)
  at Module.require (module.js:353:17)
  at require (internal/module.js:12:17)
  at Object.<anonymous> (/home/app/node_modules/gulp-coffeelint/node_modules/coffeelint/Cakefile:10:16)
  at Object.<anonymous> (/home/app/node_modules/gulp-coffeelint/node_modules/coffeelint/Cakefile:100:4)
  at Module._compile (module.js:409:26)
  at Object.exports.run (/home/app/node_modules/coffee-script/lib/coffee-script/coffee-script.js:134:23)
  at Object.exports.run (/home/app/node_modules/coffee-script/lib/coffee-script/cake.js:53:18)
  at Object.<anonymous> (/home/app/node_modules/coffee-script/bin/cake:7:38)
  at Module._compile (module.js:409:26)
  at Object.Module._extensions..js (module.js:416:10)
  at Module.load (module.js:343:32)
  at Function.Module._load (module.js:300:12)
  at Function.Module.runMain (module.js:441:10)
  at startup (node.js:134:18)
  at node.js:962:3

npm WARN This failure might be due to the use of legacy binary "node"
npm WARN For further explanations, please read
/usr/share/doc/nodejs/README.Debian
 
npm ERR! coffeelint@1.15.1 install: `cake install`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the coffeelint@1.15.1 install script.
npm ERR! This is most likely a problem with the coffeelint package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     cake install
npm ERR! You can get their info via:
npm ERR!     npm owner ls coffeelint
npm ERR! There is likely additional logging output above.
```

**Comment by swang** [comment]
Published v1.15.2 to fix this.

**Comment by paulmenzel** [comment]
Thank you for the quick reaction! It’s indeed fixed now.
